.github/workflows/scripts/scanner-update-nvd.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env python3
+
+import os
+import json
+from datetime import datetime, timedelta
+import logging
+from urllib import request, parse
+import time
+
+logging.basicConfig(
+    level=logging.INFO,
+    format="%(asctime)s: %(levelname)s: %(message)s")
+
+BASE_URL = os.getenv('SCANNER_NVD_URL', "https://services.nvd.nist.gov/rest/json/cves/2.0")
+API_KEY = os.getenv('SCANNER_NVD_API_KEY', None)
+
+
+def get_dates(year, interval):
+    if interval > 12:
+        interval = 12
+    for month in range(1, 13, interval):
+        start = datetime(year, month, 1)
+
+        # Handling for months greater than 12 (for intervals like 4, 6 months, etc.)
+        end_month = month + interval
+        end_year = year
+        if end_month > 12:
```

Comment:
why not just limit `interval` to `1, 2, 3, 4, 6, 12` (factors of 12)? That way, we don't hav to worry about overflow